index.d.ts

```diff
@@ -2665,6 +2665,8 @@ declare module 'mongoose' {
       path: string;
       reason?: NativeError | null;
       model?: any;
+
+      constructor(type: string, value: any, path: string, reason?: NativeError);
```

Comment:
There should be 1 more parameter: `schemaType?: SchemaType` . Can you add that please?

Reply:
Sure, fixed.